scripts/publish_prod.sh

```diff
@@ -41,24 +57,50 @@ rm -rf dist;
 
 #Confirm to proceed
 read -p "About to bump the version from ${CURRENT_VERSION} to ${VERSION}, and publish. Continue (y/n)?" CONT
-if ["$CONT" != "y"]; then
+if [ "$CONT" != "y" ]; then
     echo "Exiting"
     exit 1
 fi
 
-yarn login
+if git rev-parse "v${VERSION}" >/dev/null 2>&1; then
+    echo "tag v${VERSION} already exists, aborting"
+    exit 1
+fi
 
 echo "Bumping the version number and committing the changes"
 if git log --oneline -1 | grep -q "chore(release):"; then
-    echo "Create a new commit before attempting to release. Be sure to not include 'chore(release):' in the commit message, aborting"
+    echo "Create a new commit before attempting to release. Be sure to not include 'chore(release):' in the commit message. This means if the script previously prematurely ended without publishing you may need to 'git reset --hard' to a previous commit before trying again, aborting"
```

Comment:
I'm not completely sure I understand the meaning/purpose of this message, can you explain it a bit?

Reply:
@nhinsch yeah so on line 75 we use `yarn standard-version --release-as $VERSION` to bump the version number and commit the changes. As an example if someone ran the release script wanting to release version 1.0.0 then a new commit would be added with the message: `chore(release): 1.0.0`. 

So despite all the safeguards that I put in place, if for some reason the script ended prematurely after line 75 then the user wouldn't be able to run the script from beginning to end because of this logic on line 71. We want to ensure 2 releases are not made right after another, there should be some other commits between them. If someone found themselves in this situation I figured the message may help them understand.